src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml

```diff
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: 15062021_create_location_discount_table
+      author: tesliuk.ua@gmail.com
+      changes:
+        - createTable:
+            schemaName: public
+            tableName: location_discount
+            columns:
+              - column:
+                  name: loc_id
+                  type: BIGINT
+                  constraints:
+                    foreignKeyName: fk_location_discount_location
+                    references: location(loc_id)
+                    nullable: false
+              - column:
+                  name: dis_id
+                  type: BIGINT
+                  constraints:
+                    foreignKeyName: fk_location_discount_discount
+                    references: discount(dis_id)
+                    nullable: false
+              - column:
+                  name: ld_created
+                  type: TIMESTAMP
+                  defaultValueComputed: CURRENT_TIMESTAMP
+                  constraints:
+                   nullable: false
+  - changeSet:
+      id: 15062021_add_composite_pk_location_discount_table
+      author: tesliuk.ua@gmail.com
+      changes:
+        - addPrimaryKey:
+            clustered: true
+            columnNames: loc_id, dis_id
+            constraintName: pk_loc_id_dis_id
```

Comment:
rename to pk_location_discount_id

general rule for primary key names is "pk_<table_name>_id"